mkosi/__init__.py

```diff
@@ -3936,6 +3965,10 @@ def install_unified_kernel(
                     f"{prefix}/{args.machine_id}/{kver.name}",
                     "",
                 ]
+                if args.distribution == Distribution.gentoo:
+                    from .gentoo import kimg_path
+
+                    cmdline[4] = f"/usr/src/linux-{kver.name}/{kimg_path}"
```

Comment:
I dislike that this is a global. We have the Gentoo object. Why not access store this as an attribute on that?

Reply:
hm, what's the scope of our `gentoo` object? i thought it was scoped to `install_gentoo`.
or do we want to make the `gentoo` object global?

Reply:
OK, so maybe leave it as global for now, we can figure this out later.

Reply:
sure